test/ParameterStoreBuildWrapperPluginTest.groovy

```diff
@@ -84,5 +125,38 @@ class ParameterStoreBuildWrapperPluginTest {
             assertEquals(ParameterStoreBuildWrapperPlugin.class, result)
         }
     }
+
+    class withGlobalParameter {
+        @After
+        public void reset() {
+            ParameterStoreBuildWrapperPlugin.reset()
+        }
+
+        @Test
+        void addGlobalParameter() {
+            def result = ParameterStoreBuildWrapperPlugin.withGlobalParameter('/path/', [])
+
+            assertEquals([[path: '/path/']], result.globalParameterOptions)
+        }
+
+        @Test
+        void addGlobalParameterWithOptions() {
+            def result = ParameterStoreBuildWrapperPlugin.withGlobalParameter('/path/', [recursive: true, basename: 'relative'])
+
+            assertEquals([[path: '/path/', recursive: true, basename: 'relative']], result.globalParameterOptions)
+        }
```

Comment:
I like this test.  It shows that you can call `addGlobalParameters` with or without options.